tests/pass-dep/libc/libc-fs-flock.rs

```diff
@@ -0,0 +1,71 @@
+// Flock tests are separate since they don't in general work on a Windows host.
+//@ignore-target-windows: File handling is not implemented yet
```

Comment:
Was this meant to filter the host? Or is there an issue with windows targets if emulated from linux?

Reply:
No, it's about Windows targets, not hosts. This test is being executed on Windows host for Linux target.